src/main/java/de/leonhard/storage/internal/provider/LightningProviders.java

```diff
@@ -14,54 +15,54 @@
 @Accessors(fluent = true, chain = true)
 public class LightningProviders {
 
-  @Setter
-  private MapProvider mapProvider;
-  @Setter
-  private YamlConfig yamlConfig;
-  @Setter
-  private InputStreamProvider inputStreamProvider;
-  @Setter
-  private ExceptionHandler exceptionHandler;
+    @Setter
```

Comment:
If this are the only fields of this class you can also put `@Setter` on the class itself. But pay attention to the class fields, when there are some without a setter.